sig-cloud-provider/CHARTER.md

```diff
@@ -40,7 +46,7 @@ SIG Cloud Provider’s mission is to simplify, develop, and maintain cloud provi
 
 ## Roles and Organization Management
 
-This sig follows adheres to the Roles and Organization Management outlined in [sig-governance]
+This SIG follows adheres to the Roles and Organization Management outlined in [sig-governance]
```

Comment:
Do `follows` and `adheres` in this sentence kind of duplicate each other? 
```suggestion
This SIG adheres to the Roles and Organization Management outlined in [sig-governance]
```

Reply:
Ah yes, you're correct. Thanks.